ksql-rest-app/src/main/java/io/confluent/ksql/rest/util/StructSerializationModule.java

```diff
@@ -45,7 +45,7 @@ public void serialize(
         final SerializerProvider serializerProvider
     ) throws IOException {
       struct.validate();
-      jsonGenerator.writeRaw(
+      jsonGenerator.writeObject(
```

Comment:
Why are we serializing the struct as a json string?

Reply:
`jsonConverter.fromConnectData` returns `byte[]`. Using String works fine for the rest API. Do you have any other suggestion?

Reply:
If we write it as an object its going to serialize the whole struct into a json string, and then put that string into the enclosing json. That's a weird structure. It won't deserialize naturally into a nested object from other clients. For example if I write python code to read a row I'll get back a dictionary where one of the values is a string that I have to then call load on again. Its also going to render strangely in c3. You'll end up with a deeply nested structure getting dumped on one line without any nice formatting.

Reply:
What was wrong with using writeRaw?

Reply:
I changed it from String to JsonNode. This should keep the nested levels properly in c3. Writing raw wouldn't add comma when struct is an item in an array, writing as object fixes it.